From 6e2f8015a311fe3790ccd7e95d2d36eaa8080669 Mon Sep 17 00:00:00 2001
From: Li Jun <jun.li@nxp.com>
Date: Tue, 18 Jul 2017 00:51:39 +0800
Subject: [PATCH 2230/5242] MLK-16013-5 ARM64: dts: imx8mq: add
 power-down-scale property

commit  d94170b30befb8289a9438c7eaf040b686e4903c from
https://source.codeaurora.org/external/imx/linux-imx.git

As i.mx8mq USB3 suspend clock setting is 32KHz, and the default
power down scale setting is not correct, so add a property to
fix it.

Reviewed-by: Peter Chen <peter.chen@nxp.com>
Signed-off-by: Li Jun <jun.li@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
index 86efe1f..7ba4f0f 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
@@ -534,6 +534,7 @@
 			phys = <&usb3_phy0 0>, <&usb3_phy0 1>;
 			phy-names = "usb2-phy", "usb3-phy";
 			power-domains = <&power 2>;
+			snps,power-down-scale = <2>;
 			status = "disabled";
 		};
 	};
@@ -572,6 +573,7 @@
 			phys = <&usb3_phy1 0>, <&usb3_phy1 1>;
 			phy-names = "usb2-phy", "usb3-phy";
 			power-domains = <&power 3>;
+			snps,power-down-scale = <2>;
 			status = "disabled";
 		};
 	};
-- 
1.7.9.5

